infra/terraform/hash/main.tf

```diff
@@ -92,10 +92,20 @@ module "temporal" {
   vpc                   = module.networking.vpc
   env                   = local.env
   region                = local.region
-  cpu                   = 512
-  memory                = 1024
+  cpu                   = 256
+  memory                = 512
```

Comment:
We can go higher as we need to. For the current purpose it's fine to stick with the lowest spec.